srcpkgs/walk/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'walk'
+pkgname=walk
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_install_args='prefix=${DESTDIR}/usr'
+hostmakedepends="gzip make"
+short_desc="Two utilities collectively replacing find: walk and sor"
+maintainer="Andrea D'Amore <a@d-amo.re>"
+license="Apache-2.0"
+homepage="https://git.benjamin.barenblat.name/walk/"
+distfiles="https://github.com/bbarenblat/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```

Comment:
Which of the two sites is authoritative? There's no value in having one git webserver listed as a homepage and another providing distfiles. Point the homepage to the authoritative repo page and get the distfile from that.

Reply:
If the cgit site doesn't offer snapshots, then make `homepage` the GitHub repo. If the homepage is just going to be a front-end to a repository and an option to view a README, it might as well point to the actual tree you're fetching.